azure/azuredeploy.parameters.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
is it possible to specify the region that we want to spin up the cluster?

Reply:
Similarly, can we specify which subscription to use? Or is that not important?

Reply:
I dont think subscription is important.

There will be an environment variable `AZURE_REGION` if it is set, that value will be used, otherwise the default region will be used.